lumigator/frontend/src/components/molecules/LDatasetEmpty.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+  <div class="l-dataset-empty">
+    <h2 class="l-dataset-empty__instructions">Submit a dataset to start using Lumigator.</h2>
+    <p class="l-dataset-empty__instructions">
+      Please make sure you have the correct column structure.
+      <a
```

Comment:
Just a general thought, but for accessibility should we make the text inside the anchor more explicit? 

Please make sure you have the `<a>correct column structure</a>`. <-- that kind of thing.

Reply:
According to the designs this is just a suggestion.